README.md

```diff
@@ -25,14 +26,21 @@ $ poetry shell
 $ poetry install
 
 # As of 2022/12/06, `origin/fhir-updates` is a currently open development branch. It may be that `main` also has updates that haven't yet made it to release, so you could try running from that branch. If `origin/fhir-updates` is already closed/merged, you can look to see if there are any other open FHIR-related branches https://github.com/INCATools/ontology-access-kit/pulls 
-$ git checkout origin/fhir-updates
+## looks to have been merged?
```

Comment:
@chrisroederucdenver Did you mean to add this comment to the file itself, or a review comment in the GitHub PR?

That one was merged recently, and I'm on `fhir-updates-2`. If that one gets merged the next one will probably be `fhir-updates` again.

In any case I do have this line above mentioning what to do in that situation:
> If `origin/fhir-updates` is already closed/merged, you can look to see if there are any other open FHIR-related branches https://github.com/INCATools/ontology-access-kit/pulls 

---

 I'll remove these two particular lines.
```
## looks to have been merged?
##$ git checkout origin/fhir-updates
```

But I can leave this. If you want, we can also change `fhirjson` to `fhir-json`
`# careful, that's not camel case. It's a lower case j in fhirjson.`